### ISSUE TITLE
[Remove Request] - 007freepics[.]com

### ISSUE THREAD

**Issue by iam-py-test** [blocklistproject/Lists#489]
URL you wish to be removed:
`007freepics.com`

Why you believe this to be a false positive:
I can't find any evidence this site is malware; 
https://www.virustotal.com/gui/url/3e03630b2970c4f987a9eabb957e6959a1bcfa81b89b585791953b1ac9e99c03/detection
https://safeweb.norton.com/report/show?url=007freepics.com
https://www.siteadvisor.com/sitereport.html?url=007freepics.com
https://sitecheck.sucuri.net/results/007freepics.com

List it is on:
Malware

Other info you think we should know:
I think this should be added to Porn.txt

**Comment by spirillen** [comment]
I don't see any Malware reports on any of the links submitted 

No need for this in any porn lists, it's redirect to [bestfreecams.club](https://mypdns.org/my-privacy-dns/porn-records/-/issues/616)

PS: be careful where you click (crbck.link)

**Comment by iam-py-test** [comment]
> I don't see any Malware reports on any of the links submitted
> 
> No need for this in any porn lists, it's redirect to [bestfreecams.club](https://mypdns.org/my-privacy-dns/porn-records/-/issues/616)
> 
> PS: be careful where you click (crbck.link)

@spirillen that is why I made this report

**Comment by iam-py-test** [comment]
@spirillen the link you posted requires sign in

**Comment by spirillen** [comment]
Yes as also mentioned here: https://github.com/uBlockOrigin/uAssets/issues/9968#issuecomment-917678930

> Oh yeah, forgot to write about that on the login page.... its rather simple smile As I'm not interested in having mypdns.org marked as porn..... By any lists nor search engines. Therefore it is an internal repo. Just singin and you are there. It also have relation to the fact I'm unable to load a "Are 18+ to enter" script. this way you need to do something acutly to actually get acces.
>
> You should be able to use both github and gitlab oauth login api's

That said @iam-py-test then I thought this one was your account <https://mypdns.org/iam-py-test3>

**Comment by spirillen** [comment]
You should now see the following text on the login screen. https://mypdns.org/infrastructure/workboard/-/issues/22

Please add any suggestion you feel likes in the linked issue

**Comment by iam-py-test** [comment]
@spirillen yes, @iam-py-test3 is me.
I forgot about it, and I didn't have my username/password on that device

**Comment by spirillen** [comment]
> @spirillen yes, @iam-py-test3 is me.
> I forgot about it, and I didn't have my username/password on that device

LOL should I reset it for you? or maybe match your profile name to `iam-py-test`?

Can you remember the mail you used?

**Comment by iam-py-test** [comment]
> 
> 
> > @spirillen yes, @iam-py-test3 is me.
> > I forgot about it, and I didn't have my username/password on that device
> 
> LOL should I reset it for you? or maybe match your profile name to `iam-py-test`?
> 
> Can you remember the mail you used?

No. I have it saved
I just switch devices and browsers a lot, so I don't have all my passwords saved everywhere
